sale_automatic_workflow_payment_mode/tests/test_automatic_workflow_payment_mode.py

```diff
@@ -96,5 +96,3 @@ def test_full_automatic(self):
         )
         self.env["automatic.workflow.job"].run()
         self.assertEqual(invoice.payment_state, "paid")
-        picking = sale.picking_ids
-        self.assertEqual(picking.state, "done")
```

Comment:
@john-herholz-dt Why removing this test?

Reply:
@rousseldenis 
It was seperated in 17.0
sale_automatic_workflow is no longer validating the picking.
There is a new module sale_automatic_workflow_stock

Also, this module should not be responsible for checking something in the pickings - as it has no logic (code) for this.
Or am I wrong?